src/configprovider-lldb.ts

```diff
@@ -0,0 +1,65 @@
+import * as vscode from "vscode";
+import * as path from "path";
+import { getMesonTargets } from "./introspection";
+import { Target } from "./types";
+import { extensionConfiguration, getTargetName } from "./utils";
+
+export class DebugConfigurationProviderLldb implements vscode.DebugConfigurationProvider {
```

Comment:
How much of this class is now a copy-paste of the Cppdbg one? Anyway that we could factor the copy-paste into a common abstract base class like "MesonDebugConfigurationProvider"?

Reply:
Quite a lot apparently. e817bb1 should fix things.